plugin/mode_indicator/mode_indicator.py

```diff
@@ -115,6 +123,22 @@ def on_draw(c: SkiaCanvas):
     c.paint.color = color_mode
     c.draw_circle(x, y, radius)
 
+    if settings.get("user.mode_indicator_show_microphone_name"):
+        # Remove c.paint.shader gradient before drawing again
+        c.paint.shader = skia.Shader.radial_gradient(
+            Point2d(x, y), radius, [color_text, color_text]
+        )
+
+        text = actions.sound.active_microphone()[:2]
```

Comment:
```suggestion
        text = current_microphone[:2]
```
Just realized that we are already reading this action and caching the result.